modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/test/js/setup.es.js

```diff
@@ -13,6 +13,11 @@ const APIUtil = require('../../src/main/resources/META-INF/resources/js/utils/ap
 const LanguageUtil = require('../../src/main/resources/META-INF/resources/js/utils/language.es');
 import {FETCH_HIDDEN_DOCUMENTS_URL, getMockResultsData} from './mocks/data.es';
 
+/**
+ * Extends the timeout session due to several async callbacks not being invoked
+ */
```

Comment:
Can we also include the exact error message we're attempting to prevent?
```Timeout - Async callback was not invoked within the 5000ms timeout specified by jest.setTimeout.Error```

That way it makes it clear what issue we're trying to solve which will allow us to confidently improve/remove in the future.

Reply:
Updated! Let me know if it needs revision